src/parser/syntax.ml

```diff
@@ -1227,8 +1236,10 @@ and explicit_list_items :
     let next_token = peek input in
     match next_token.value with
     | `End ->
-        Parse_error.not_allowed next_token.location ~what:(Token.describe `End)
+        Parse_error.end_not_allowed next_token.location
+          ~what:(Token.describe `End)
           ~in_what:(Token.describe parent_markup)
+          ~suggestion:"try to add '}' at the end of the list content."
```

Comment:
I hoped that we wouldn't need to pass `~what` and `~suggestion`.

Reply:
I hoped to use the same error function -- `end_not_allowed` for both unclosed table and list, I thought it would be important to specify the markup that is not closed (which is what the `~what` is for). But I now see that it's not important, I will remove that.

Reply:
I think `~what` is important